### PR TITLE
[cleanup] Fix warnings, typos, renames

### PR DESCRIPTION
**Pull request by falkecarlsen**
Made as a response to this: https://github.com/d610f20/random-activitygen-py/pull/26#pullrequestreview-389090158.

Also ran inspection on codebase and fixed all warnings.

### CODE REVIEW

**Review by MathiasMehl — CHANGES_REQUESTED** [review]
Could add some logging to `setup_bus_stops`, otherwise looks good

**Comment by falkecarlsen** [comment]
> Could add some logging to `setup_bus_stops`, otherwise looks good

Gotcha 232833f :ok_hand:

**Review by MathiasMehl — APPROVED** [review]

**Review by jenrik — APPROVED** [review]

File: perlin.py [hunk 1]
```diff
@@ -46,12 +46,12 @@ def get_perlin_noise(x: float, y: float, base: int, scale: float = 0.005, octave
     return (noise.pnoise3(x=x * scale, y=y * scale, z=base, octaves=octaves) + 1) / 2
 
 
-def get_population_number(edge: sumolib.net.edge.Edge, base: int, centre,
-                          radius, centre_weight: float = 1.0, scale: float = 0.005, octaves: int = 3) -> float:
+def get_noise_interval(edge: sumolib.net.edge.Edge, base: int, centre,
```
Comment (NicEastvillage) [inline perlin.py]: It doesn't return an interval. Alternatives:
`get_noise_sample`
`sample_edge_noise`

**Review by NicEastvillage — APPROVED** [review]

File: perlin.py (hunk 1 above)
Comment (falkecarlsen) [reply perlin.py]: Fixed in d4c691d

**Pull request merged**